### ISSUE TITLE
ValueError: Invalid value for `additional`

### ISSUE THREAD

**Issue by sim0nx** [rapid7/vm-console-client-python#5]
## Issues and Contributing
Using the following code to get the configured reports:
```python
client = rapid7vmconsole.ApiClient(configuration=config)
rapid7vmconsole.ReportApi(client).get_reports()
```

gives the following error:

> ValueError: Invalid value for `additional` (email), must be one of ['file', 'zip', 'none']

There is exactly one report and indeed that report has configured some additional non-members with just some e-mail addresses.
This was configured in nexpose directly and is working as expected.

### Expected Behavior
The python API should accept "email" as valid additional option.

### Actual Behavior
It fails with:
> ValueError: Invalid value for `additional` (email), must be one of ['file', 'zip', 'none']

### Possible Solution
Add "email" as valid type in file "rapid7vmconsole/models/report_email.py" on line 126.

### Steps to Reproduce (for bugs)
Configure a report with additional email recipients, with custom addresses (not users).

Python code that reproduces the issue:
```python
client = rapid7vmconsole.ApiClient(configuration=config)
rapid7vmconsole.ReportApi(client).get_reports()
```

### Your Environment
* Library version or repository branch:   rapid7-vm-console     0.0.1-6.5.19
* Python version:   3.6.4
* Operating System and version:  Debian 9
* InsightVM/Nexpose product version:  nexpose 6.5.24

**Comment by sim0nx** [comment]
I can confirm that applying the suggested modification fixes the problem

**Comment by rwmanos** [comment]
I experienced the same issue and I came up with the exact same solution! I was ready to report it and create a PR but I saw @pranava-b had done it already (why closed the PR though? https://github.com/rapid7/vm-console-client-python/pull/10 )

The real problem is that the output of https://help.rapid7.com/insightvm/en-us/api/api.json contains:
```
additional":{"type":"string","example":"file","description":"The format to distribute the report to additional recipients.","enum":["file","zip","none"]}
```
which creates this bug in the client.

I've submitted a ticket with Rapid7 and hopefully someone will fix it.

**Comment by rwmanos** [comment]
@sim0nx FYI, this has been fixed with the release of version 6.5.65 on 2019-05-29.
The release notes contain:
> We fixed an inconsistency with the "Additional" recipient field for report distribution in APIv3.

The fix does not include pranava-b's proposed patch but instead corrected the issue on the back-end of the API. The Java code was taking in 'email' as a valid field when it should have been using 'file' as the valid field.

I checked and I cannot reproduce the bug. Check it as well if you want and feel free to close the issue.
